kbs/src/api/src/http/resource.rs

```diff
@@ -19,6 +19,11 @@ use crate::raise_error;
 
 use super::*;
 
+#[cfg(feature = "as")]
+const TOKEN_TEE_PUBKEY_PATH: &str = AS_TOKEN_TEE_PUBKEY_PATH;
+#[cfg(not(feature = "as"))]
+const TOKEN_TEE_PUBKEY_PATH: &str = "/customized_claims/runtime_data/tee-pubkey";
```

Comment:
There are two places defining TOKEN_TEE_PUBKEY_PATH, like [before](https://github.com/confidential-containers/trustee/pull/414/files#diff-b83a2a4e4135ca8f4193335a1a8d660b5f1dadf86f8cad301b44c4f2a6867a2eR20). Is it possible to define only on one place and use a reference on the other side?

Reply:
I wanted it to be like that but I found the problem that `TOKEN_TEE_PUBKEY_PATH` needs to be defined even completely without Attest  (i.e., `as` not enabled) so some "default" must be kept here.